spec/sections/07-conventions.adoc

```diff
@@ -57,6 +57,12 @@ The following IRI namespace prefixes are used throughout this document:
 
 All of these namespace prefixes in the previous section resolve to resources that contain their namespace content except for `eg:` (`http://example.com/`), which is used just for examples, and `ogc:` (`http://www.opengis.net/`), which is used in requirement specifications as a placeholder for the geometry literal serialization used in a fully-qualified conformance class, e.g. http://www.opengis.net/ont/geosparql#wktLiteral[`+<http://www.opengis.net/ont/geosparql#wktLiteral>+`].
 
+=== Data Types for Spatial Aggregate Functions
+
+In this specification we use the placeholder URI ogc:geomLiteral to describe any geometry literal which is defined in this specification.
+To express a list of such literals, for example as input parameters for spatial aggregate functions, we use the notation ogc:geomLiteral[].
```

Comment:
don't you need to put ogc:geomLiteral[] in some code markers? (I'm not familiar with adoc)